index.bs

```diff
@@ -168,6 +173,34 @@ Example:
   JSON.stringify(Sanitizer.defaultConfig()) == JSON.stringify(new Sanitizer().config());  // true
 ```
 
+## Presets ## {#presets}
+
+The value of each of the static preset attributes, {{default}}, {{nofetch}},
+{{nonavigate}}, and {{richtext}} is the result of creating a
+{{Sanitizer}} instance with the appropriate configuration constant used as the
+[=configuration object=]. The pairing of {{Sanitizer}} preset attributes
+and the constants to use as the [=configuration object=] are listed below:
+
+* <dfn attribute for=Sanitizer>default</dfn>: [=default configuration=] object.
+* <dfn attribute for=Sanitizer>nofetch</dfn>: [=nofetch configuration object=].
+* <dfn attribute for=Sanitizer>nonavigate</dfn>: [=nonavigate configuration object=].
+* <dfn attribute for=Sanitizer>richtext</dfn>: [=richtext configuration object=].
+
+Examples:
+```js
+  // Preset "rich text" allows only formatting:
+  // <p class="blubb"><b>text</b></p>
+  Sanitizer.richtext.sanitize("<p id=bla class=blubb><a href=https://example.org><b>text</b></a>");
```

Comment:
With my suggestion above someone would still have to initialize it if they need it.
e.g., `myCommonMarkSanitizer = new Sanitizer(Sanitizer.CONFIG_COMMONMARK)`